README.md

```diff
@@ -71,6 +71,24 @@ conda install qtpy
 ```
 
 
+### mypy
+
+A CLI is offered to help with usage of QtPy including the subcommand
+`qtpy mypy args` which generates command line arguments similar to the
+following which help guide mypy through which library QtPy would have used
+so that mypy can get the proper underlying type hints.
+
+```
+--always-false=PYQT4 --always-false=PYQT5 --always-false=PYSIDE --always-true=PYSIDE2
+```
+
+Use such as:
```

Comment:
I guess my intention was that its better to be explicit rather than implicit about what users are expected to replace, if the example is intended to be of practical value. However, given your point, perhaps its simpler to just specify the default invocations of the `mypy` and `qtpy` installed in the current environment (i.e. `mypy` and `qtpy`), which users can replace themselves with someone more specific if their situation requires it? Or is there some particular reason that `mypy` needs to be invoked with the full path?